hathor/feature_activation/resources/feature.py

```diff
@@ -160,7 +160,23 @@ class GetFeaturesResponse(Response):
 
 FeatureResource.openapi = {
     '/feature': {
-        'x-visibility': 'private',
+        'x-visibility': 'public',
+        'x-rate-limit': {
+            'global': [
+                {
+                    'rate': '50r/s',
+                    'burst': 100,
+                    'delay': 50
+                }
+            ],
+            'per-ip': [
+                {
+                    'rate': '3r/s',
```

Comment:
That much? Maybe 1r/s?

Reply:
Sure, changed in https://github.com/HathorNetwork/hathor-core/pull/743/commits/0d44bb5217e53a7728897aa36533a4f58dafc7f7